modules/editor-service/src/main/java/org/opencastproject/editor/EditorServiceImpl.java

```diff
@@ -972,7 +1011,7 @@ public EditingData getEditData(final String mediaPackageId) throws EditorService
             .collect(Collectors.toList());
 
     return new EditingData(segments, tracks, workflows, mp.getDuration(), mp.getTitle(), event.getRecordingStartDate(),
-            event.getSeriesId(), event.getSeriesName(), workflowActive, waveformList, subtitles);
+            event.getSeriesId(), event.getSeriesName(), workflowActive, waveformList, subtitles, localPublication);
```

Comment:
This use of `localPublication` is dependent on the state of the *last* call to `isLocal`.  This is *probably* fine, but what if one file isn't local somehow, and the rest are?

Reply:
Then, you don't have a shared file system but made a hell of mess on your server since you manually copied your distribution files to other servers where they aren't used. For me, that's a bit like asking, “but what if someone manually wrote stuff into the database”? The answer: Things break. However, you can configure this away from `auto` in these cases if you want to.

We could check, but to be sure, we would need to check all the files and then, things might still break if you find and flag one as not-local because of https://github.com/opencast/opencast/issues/3538

Reply:
Yeah, ok fair enough.  I could see this biting us in the future if we somehow do tiered storage for delivery, but that's a problem for future us and can be addressed at that point.